lib/logger.js

```diff
@@ -4,27 +4,42 @@
  * If for any reason, they need a different logging tool or need to be extended,
  * the changes can be made here.
  */
-var mergeOptions = require('./get-options').mergeOptions;
+var getOptions = require('./get-options');
 var util = require('util');
-var utility = require('./utility');
 var winston = require('winston');
 
 /**
- * Abstracts fully-fleshed logging module to the logger property of this module.
+ * Get options for logging system. Checks project's options against defaults, and sets up Winston's transport requirements
  *
- * If logging with metadata, or requiring winston.log's more advanced features,
- * use exports.logger().log() instead of exports.log().
- * @example
- *   exports.logger().log('info', 'Test Log Message', { anything: 'This is metadata' });
- *
- * @return {Object} An instance of winston.Logger.
+ * @return {Object} Contains Winston-specific options for transports and exiting on errors.
  */
-exports.logger = function () {
+function getLoggerOptions() {
   'use strict';
 
-  var options = utility.getProjectOptions();
+  var defaultOptions = {
+    logger: {
+      level: 'info',
+      silent: false,
+      exitOnError: true,
+      logFile: ''
+    }
+  };
+
+  var loggerOptions = {};
+
+  // get the project's options
+  var projectOptions = getOptions.getProjectOptions();
 
-  var transports = [
+  // merge project options with default options; project options take precedence
+  var options = getOptions.mergeOptions(defaultOptions, projectOptions);
+
+  // check if we're unit testing
+  if (isTest()) {
+    options.logger.silent = true;
+  }
```

Comment:
@scottnath This refactor will suppress errors and warnings in tests. I don't think this is what we want.